demisto_sdk/commands/download/downloader.py

```diff
@@ -597,7 +608,7 @@ def update_data(file_path_to_write: str, file_path_to_read: str, file_ending: st
         :return: None
         """
         ryaml = YAML()
-        ryaml.preserve_quotes = True
+        ryaml.preserve_quotes = True  # type: ignore
```

Comment:
why?

Reply:
because of ryaml annotations is faulty. (`preserve_quotes` Is None and not bool)